manifests/0000_05_config-operator_01_console.crd.yaml

```diff
@@ -16,3 +16,14 @@ spec:
   - name: v1
     served: true
     storage: true
+  validation:
+    openAPIV3Schema:
+      properties:
+        spec:
+          properties:
+            authentication:
+              properties:
+                logoutRedirect:
+                  pattern: ^$|^((https):\/\/?)[^\s()<>]+(?:\([\w\d]+\)|([^[:punct:]\s]|\/?))$
```

Comment:
Testing this with a regex tester:

```
https://foobar                              // valid, probably should not be?
https://redhat.com/#foo             // valid, i dont think we care about fragments
https://redhat.com?red=hat       // valid, i dont think we care about query strings
```

already stated in prev comment:

```
https://www.redhat.com:8080/logout.html  // valid
http://www.redhat.com                                   // invalid
www.redhat.com is invalid                              // invalid
```

Reply:
> `https://foobar                              // valid, probably should not be?`

Is there a reason we would require a path? I'd think this is OK

Reply:
I wanted https://localhost to work

Reply:
@spadgett I was more commenting on the `.<something>`.  
`localhost` does seem a valid point.